src/keyboard/keyboardImplementation.ts

```diff
@@ -48,11 +50,20 @@ export async function keyboardImplementation(
         keyDef,
         getCurrentElement,
         options,
-        state,
+        fireKeyDownTimes > (state.fireKeyDownTimes || 1)
+          ? {...state, fireKeyDownTimes}
```

Comment:
Please don't create a derived `state`.
The state should be a shared truth about the state of the input device(s) across all our events.

Reply:
Reverted 👍🏻